statefun-flink/statefun-flink-core/src/main/java/org/apache/flink/statefun/flink/core/state/FlinkTableAccessor.java

```diff
@@ -55,4 +57,19 @@ public void remove(K key) {
       throw new RuntimeException(e);
     }
   }
+
+  @Override
+  public Iterable<Map.Entry<K, V>> view() {
+    try {
+      Iterable<Map.Entry<K, V>> entries = handle.entries();
+      return entries == null ? Collections.emptyList() : entries;
```

Comment:
Actually, I think Flink user `MapState` never returns `null` for `entries()` / `keys()` / `values()`.

The actual `MapState` handle that the user gets is always a `UserFacingMapState` in Flink that wraps the actual RocksDB / heap map state handle, which uses an empty map state if the wrapped state handle returns `null` for those methods.

Reply:
Looks like I have some Flink jobs I can remove some null checks from :)